source/simulator/helper_functions.cc

```diff
@@ -215,6 +223,71 @@ namespace aspect
   }
 
 
+
+  template <int dim>
+  void Simulator<dim>::write_plugin_graph (std::ostream &out) const
+  {
+    // write the preamble
+    out << "digraph Plugins\n"
+        "{\n"
+        "  splines=line;\n"
+        "  splines=true;\n"
+        "  overlap=false;\n"
```

Comment:
Ah. Just use the `neato` command line tool. I could also add the layout statement if you want.